tests/expectations/metrics/test_core.py

```diff
@@ -1703,6 +1703,75 @@ def test_map_column_pairs_in_set_metric_pd():
     )
 
 
+def test_map_column_pairs_in_set_metric_sa(sa):
+    engine = build_sa_engine(
+        pd.DataFrame({"a": [10, 3, 4, None, 3, None], "b": [1, 2, 3, None, 3, 5]}), sa
+    )
+
+    metrics: dict = {}
+
+    table_columns_metric: MetricConfiguration
+    results: dict
```

Comment:
We use this annotation a decent bit in our codebase but would it be possible to specify the types of the keys and values here? I only ask because this defaults to `Dict[Any, Any]` and any additional information you know about the object could prove to be quite valuable for future users of the code.